python/lsst/ts/cRIOpy/GUI/CustomLabels.py

```diff
@@ -70,6 +71,44 @@ def __init__(self):
         self.setFrameShadow(QFrame.Sunken)
 
 
+class DataLabel(QLabel):
+    """Displays data from (SAL originated) signal.
+
+    Parameters
+    ----------
+    signal : `Signal`, optional
+        When not None, given signal will be connected to method calling
+        setValue with a field from signal data. Field is the second argument.
+        Defaults to None.
```

Comment:
"Defaults" or "Default"?

Reply:
"Defaults to" is now used in document. "Default to" is also an option, I can change it - but so far I found the both forms are possible.